practice_05.11.19_classes/Task2_Plants/Plant.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+namespace practice_05._11._19_classes.Task2_Plants
+{
+    public class Plant
+    {
+        public string Type { get; set; }
```

Comment:
I think type of a plant should not be able to change type after it is created.

Reply:
fixed